### PR TITLE
fortify-on-demand-mobile-assessment 0.1.0

### PR DESCRIPTION
**Pull request by jeremy-chua**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- opts:
+    is required: true
+    is_expand: true
+    summary: Platform type for this mobile assessment
+    title: Platform Type
+    value_options:
+    - Phone
+    - Tablet
+    - Both
+  platform_type: ' '
```
Comment (koral--) [inline steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:112]: This value is outside the option list

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- client_secret: null
+  opts:
+    description: Authenticating with Client Credentials. The client credentials are
+      the API keys generated in the portal.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    summary: Client secret for authentication
+    title: Client Secret
+- datacenter: ' '
```
Comment (Bence1001) [inline steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:49]: Default value must be among `value_options`.

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    title: Entitlement ID
+- opts:
+    description: The ID assigned to the created release. This value can be reterived
+      in Foritfy on Demand portal.
+    is_expand: true
+    is_required: true
+    summary: Release ID for this mobile assessment
+    title: Release ID
+  release_id: null
+- assessment_type: ' '
```
Comment (Bence1001) [inline steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:78]: Default value must be among `value_options`.

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    is required: true
+    is_expand: true
+    summary: Assessment type for this mobile assessment
+    title: Assessment Type
+    value_options:
+    - Mobile Assessment (Single Scan)
+    - Mobile+ Assessment (Single Scan)
+    - Mobile Assessment (Subscription)
+    - Mobile+ Assessment (Subscription)
+- framework_type: ' '
```
Comment (Bence1001) [inline steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:94]: Default value must be among `value_options`.

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml (hunk 1 above)
Comment (jeremy-chua) [reply steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:112]: I did not make any changes in step.yml
It was approved in previous versions 0.0.1 and 0.0.2
You can refer to 
https://github.com/bitrise-io/bitrise-steplib/pull/3225/files/9be8fbbd2d811f389e5243f4b9aa6b677ff51ef8

File: steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml (hunk 4 above)
Comment (jeremy-chua) [reply steps/fortify-on-demand-mobile-assessment/0.1.0/step.yml:94]: I did not make any changes in step.yml
It was approved in previous versions 0.0.1 and 0.0.2
You can refer to 
https://github.com/bitrise-io/bitrise-steplib/pull/3225/files/9be8fbbd2d811f389e5243f4b9aa6b677ff51ef8

**Pull request closed without merging**